src/run.js

```diff
@@ -164,10 +155,11 @@ const processPage = ({
       }
 
       // A must or else you can't do console.log from within page.evaluate()
-      page.on('console', msg => {
+      page.on('console', async (msg) => {
         if (debug) {
-          for (let i = 0; i < msg.args.length; ++i) {
-            console.log(`${i}: ${msg.args[i]}`);
```

Comment:
maybe this worked in older versions in puppeteer, but it doesn't work in recent versions. I guess we also need to bump puppeteer version